doc/string/split.rdoc

```diff
@@ -12,7 +12,7 @@ When +field_sep+ is <tt>$;</tt>:
   the split occurs just as if +field_sep+ were given as that string
   (see below).
 
-When +field_sep+ is <tt>' '</tt> and +limit+ is +nil+,
+When +field_sep+ is <tt>' '</tt> and +limit+ is omitted,
```

Comment:
I think we should change `omitted` to `0`, both here and in other cases.  When `limit` is omitted, it has the default value of `0`, which is why it works. We should only mention behavior with omission of arguments if the behavior is different than when passing the default argument values explicitly.

Reply:
Thank you for your review. PR fixed.